lib/ProductOpener/Store.pm

```diff
@@ -369,15 +391,18 @@ sub link_object($path, $link) {
 	symlink($path . '.json', $link . '.json')
 		or $log->error("could not link", {source => $path, link => $link, error => $!});
 
+	# We normally delete a link before creating a new one but just in case make sure there is no STO link
+	unlink($link . '.sto');
```

Comment:
Would it be worth an error/warning message here if the operation is successful?  (e.g. unexpected leftover `.sto` file found, and was deleted)